freeipa.tf

```diff
@@ -29,6 +29,16 @@ locals {
 
   # The subnets where the IPA servers are to be placed
   subnet_cidrs = keys(data.terraform_remote_state.networking.outputs.private_subnets)
+
+  # The IP addresses of the IPA servers.  AWS reserves the first four
+  # and the last IP address in each subnet.
+  #
+  # The latest version of cisagov/freeipa-server-tf-module requires us
```

Comment:
`The latest version of` should probably be removed from this comment, as the current "latest version" will not be so for long.

Reply:
See commit ba56ca2.